app/controllers/application_controller/ci_processing.rb

```diff
@@ -1619,6 +1619,42 @@ def scanhosts
     host_button_operation('scan', 'Analysis')
   end
 
+  # Scan all selected or single displayed image(s)
+  def scanimages
```

Comment:
I agree with @abonas here ... I glanced at the method names in `CiProcessing` and it looks like about a 50/50 split on methods with unconventional names and methods with conventional names.

I'd go with the conventional name even if it's surrounded by unconventional names in the same file.